docs/custom_rules.md

```diff
@@ -0,0 +1,76 @@
+# Custom Rules
+The Custom Rules feature allows any customer to create simple rules using a pre-defined set of operators. These rules support any resource property comparisons.
+
+# Syntax
+Each rule has been designed to be easy to write, and thus the syntax is very flexible in regards to grammar. Each rule is always one line long, and follows a set structure in regards to syntax.
+
+The template used for each custom rules have been included below. Angle brackets specify required values, while square brackets indicate optional values.
+
+`<Resource Type> <Property[*]> <Operator> <Value> [Error Level] [Custom Error Message]`
+
+
+### General Guidelines
+
+* The ruleID is auto-generated based on the line number. The E9XXX and W9XXX blocks are allocated towards custom rules.
+    * As an example, custom rule on line 4 of the rules file which has error level “ERROR” would become E9004
+* Comments are supported through the use of the # symbol at the beginning of a line (e.g `#This is a comment`)
+* The syntax is "quote-flexible" and will support all permutations shown below
+    * Valid: AWS::Serverless::Function Property EQUALS "Cloud Formation"
+    * Valid: AWS::Serverless::Function Property EQUALS Cloud Formation
+    * Valid: AWS::Serverless::Function Property EQUALS Cloud Formation WARN "Custom Error"
+    * Valid: AWS::Serverless::Function Property EQUALS Cloud Formation WARN Custom Error
```

Comment:
Nitpick, but [AWS::Serverless resource types](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-specification-resources-and-properties.html) are probably weird examples because [they should be transformed out of the template before linting is run](https://github.com/aws-cloudformation/cfn-python-lint/#serverless-application-model)

Reply:
Applied in next fix, example changed to EC2 instances